scripts/Dockerfile

```diff
@@ -1,18 +1,55 @@
-FROM jupyter/minimal-notebook
+FROM jupyter/base-notebook:python-3.9.12
 
-ARG SRC_DIR=.
+LABEL name="pymc"
+LABEL description="Environment for PyMC version 4"
 
-MAINTAINER Austin Rochford <austin.rochford@gmail.com>
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 
-ADD $SRC_DIR /home/jovyan/
-RUN /bin/bash /home/jovyan/scripts/create_testenv.sh --global --no-setup
+USER root
```

Comment:
Containers should not run as root. Is there a reason this is needed?

https://theorangeone.net/posts/containers-as-root/

Reply:
Hi agree, it has been removed :)